### PR TITLE
@kanaabe => Revise mobile inject code / Display behavior

### PR DESCRIPTION
**Pull request by damassi**
Addresses https://github.com/artsy/publishing/issues/192 and fixes issue where sidebar display that was re-positioned after the second paragraph (on mobile) was not receiving events.

### CODE REVIEW

File: src/Components/Publishing/Display/__test__/DisplayPanel.test.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+      const instance = wrapper.instance()
+      instance.isVideoClickArea = () => true
+      instance.video = { play: jest.fn(), pause: jest.fn() }
+      const spy = jest.spyOn(instance, 'onClickVideo')
+      wrapper.update()
+      wrapper.find('DisplayPanelContainer').simulate('click')
+      expect(spy).toHaveBeenCalled()
+      expect(wrapper.state().isPlaying).toEqual(true)
+      wrapper.find('DisplayPanelContainer').simulate('click')
+      expect(wrapper.state().isPlaying).toEqual(false)
```
Comment (damassi) [inline src/Components/Publishing/Display/__test__/DisplayPanel.test.js]: @eessex - a lot of the testing things we talked about the other day are in this block. Unless anyone has any issues with this, note that I named the file `.js` to get around typescript errors related to stubbing / overriding methods.

File: src/Components/Publishing/Display/__test__/DisplayPanel.test.js [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+      instance.isVideoClickArea = () => true
+      const spy = jest.spyOn(instance, 'onClickVideo')
+      const toggleSpy = jest.spyOn(instance, 'toggleVideo')
+      wrapper.update()
+      wrapper.find('DisplayPanelContainer').simulate('click')
+      expect(spy).toHaveBeenCalled()
+      expect(toggleSpy).toHaveBeenCalled()
+      expect(wrapper.state().isPlaying).toEqual(true)
+      wrapper.find('DisplayPanelContainer').simulate('click')
+      expect(wrapper.state().isPlaying).toEqual(false)
```
Comment (damassi) [inline src/Components/Publishing/Display/__test__/DisplayPanel.test.js]: @eessex - a lot of the testing things we talked about the other day are in this block. Unless anyone has any issues with this, note that I renamed the file extension to `.js` to get around typescript errors related to stubbing / overriding methods.

File: src/Components/Publishing/Display/__test__/DisplayPanel.test.jsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+      instance.isVideoClickArea = () => true
+      const spy = jest.spyOn(instance, 'onClickVideo')
+      const toggleSpy = jest.spyOn(instance, 'toggleVideo')
+      wrapper.update()
+      wrapper.find('DisplayPanelContainer').simulate('click')
+      expect(spy).toHaveBeenCalled()
+      expect(toggleSpy).toHaveBeenCalled()
+      expect(wrapper.state().isPlaying).toEqual(true)
+      wrapper.find('DisplayPanelContainer').simulate('click')
+      expect(wrapper.state().isPlaying).toEqual(false)
```
Comment (damassi) [inline src/Components/Publishing/Display/__test__/DisplayPanel.test.jsx]: @eessex - a lot of the testing things we talked about the other day are in this block. Unless anyone has any issues with this, note that I renamed the file extension to `.js` to get around typescript errors related to stubbing / overriding methods.

**Comment by damassi** [comment]
@kanaabe / @eessex - i'm going to self-merge to get this into staging testing. Will continue covering behaviors in tests as I come across them.

File: src/Components/Publishing/Display/__test__/DisplayPanel.test.jsx (hunk 3 above)
Comment (damassi) [reply src/Components/Publishing/Display/__test__/DisplayPanel.test.jsx]: Also note that to query by styled-component container similar to ` wrapper.find('DisplayPanelContainer').simulate('click')`, you've gotta name it 

```jsx
const DisplayPanelContainer = styled.div` ... `
DisplayPanelContainer.displayName = 'DisplayPanelContainer'
```

File: src/Components/Publishing/Display/DisplayPanel.tsx [hunk 4]
```diff
@@ -234,6 +277,7 @@ const DisplayPanelContainer = Div`
     margin: auto;
   `}
 `
+DisplayPanelContainer.displayName = "DisplayPanelContainer"
```
Comment (kanaabe) [inline src/Components/Publishing/Display/DisplayPanel.tsx]: Seeing a duplicate of this line below --

File: src/Components/Publishing/Display/__test__/DisplayPanel.test.jsx [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+      instance.isVideoClickArea = () => true
+      instance.video = { play: jest.fn(), pause: jest.fn() }
+      wrapper.find('DisplayPanelContainer').simulate('mouseEnter')
+      expect(wrapper.state().isPlaying).toEqual(true)
+      wrapper.find('DisplayPanelContainer').simulate('mouseLeave')
+      expect(wrapper.state().isPlaying).toEqual(false)
+    })
+  })
+})
+
```
Comment (kanaabe) [inline src/Components/Publishing/Display/__test__/DisplayPanel.test.jsx]: Nice set of unit tests!

File: src/Components/Publishing/Display/DisplayPanel.tsx (hunk 4 above)
Comment (damassi) [reply src/Components/Publishing/Display/DisplayPanel.tsx]: This was fixed in https://github.com/artsy/reaction/pull/361

**Pull request merged**